mappings/net/minecraft/client/render/tile/TileEntityRenderDispatcher.mapping

```diff
@@ -0,0 +1,39 @@
+CLASS net/minecraft/class_331 net/minecraft/client/render/tile/TileEntityRenderDispatcher
+	FIELD field_1552 INSTANCE Lnet/minecraft/class_331;
+	FIELD field_1553 renderOffsetX D
+	FIELD field_1554 renderOffsetY D
+	FIELD field_1555 renderOffsetZ D
+	FIELD field_1556 textureManager Lnet/minecraft/class_76;
+	FIELD field_1557 level Lnet/minecraft/class_18;
+	FIELD field_1558 cameraEntity Lnet/minecraft/class_127;
+	FIELD field_1559 yaw F
+	FIELD field_1560 pitch F
+	FIELD field_1561 x D
+	FIELD field_1562 y D
+	FIELD field_1563 z D
+	FIELD field_1564 renderers Ljava/util/Map;
+	FIELD field_1565 textRenderer Lnet/minecraft/class_34;
```

Comment:
font was a funi name smh
on a serious note, I assume you changed it to match the actual name we gave the class
might be funny and cool to ppush to plasma and change all instances to font :yotef*kinhaw:

Reply:
:b:ad